src/components/Charts.jsx

```diff
@@ -2,52 +2,51 @@
 
 import React from 'react';
 
+import Immutable from 'immutable';
+
 import ChartWithHeader from './ChartWithHeader';
 
+const DIV_STYLE = {overflow: "auto", maxHeight: "100vh"};
+
 class Charts extends React.Component {
     render() {
-        var charts = [];
+        let charts = [];
         if (this.props.selectedVersions !== undefined && this.props.dataSets !== undefined) {
```

Comment:
This evaluates to true also if both are `null`, just saying. That's basically the only okay usage of `!=`, to check for `null` or `undefined`.